src/main/java/com/thealgorithms/backtracking/Combination.java

```diff
@@ -1,62 +1,58 @@
 package com.thealgorithms.backtracking;
 
-import java.util.Arrays;
-import java.util.LinkedList;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.TreeSet;
 
 /**
- * Finds all permutations of given array
- * @author Alan Piao (<a href="https://github.com/cpiao3">git-Alan Piao</a>)
+ * Finds all combinations of a given array of unique elements.
```

Comment:
This is not verified anywhere. Maybe the input type should be `Set<T>`? Or maybe this statement is not needed. I think what we are aiming for it to mimic the [`itertools.combinations`](https://docs.python.org/3/library/itertools.html#itertools.combinations) from python.